### PR TITLE
Fixes "General Information" section of CV in dark mode.

### PR DESCRIPTION
**Pull request by jkschin**
## Problem

When the dark mode is default, or if you set the page to dark mode and then refresh, the content in "General Information" can't be seen.

Screen Shot 2023-05-18 at 10 46 23 AM

## Solution

This PR fixes the problem.

### CODE REVIEW

File: assets/js/theme.js
Comment (alshedivat) [inline assets/js/theme.js:86]: if I understand correctly, this is the main change. everything else is small refactors?

Comment (alshedivat) [reply assets/js/theme.js:86]: actually, this change seems to introduce flickering effect: when you switch to dark mode and then start clicking on other pages, it first loads the new page in the light mode and then switches it to dark mode, creating a weird transition.

**Review by alshedivat — CHANGES_REQUESTED** [review]
please fix the flickering effect, and I'll be happy to merge.

File: assets/js/theme.js
Comment (jkschin) [reply assets/js/theme.js:86]: Yup. The other refactors were proposed by Prettier. Good catch on the flickering effect. Let me see how I can fix it.

**Comment by jkschin** [comment]
@alshedivat I've posted a nicer fix. I did two things

1. I set the background of the table in `map.html` to transparent.
2. I set the color to follow the `global-text-color`.

This makes it look better now, without the flickering effect too.

File: _includes/cv/map.html
Comment (alshedivat) [inline _includes/cv/map.html:1]: do these styles have to be in-line or you can move them to `_sass/base.scss`?

**Review by alshedivat — APPROVED** [review]
LGTM, thanks again for the fix!

File: _includes/cv/map.html
Comment (jkschin) [reply _includes/cv/map.html:1]: I think the classes like `table`, `table-sm`, etc. might be used in more than one place. I wasn't sure if this effect is something we want to apply globally. Tables could also appear in posts or other places.

Intuitively, it should be what we want since people who use the exact tables in dark mode would probably see nothing? But I can't be sure. I'll defer to your judgment on this.

Comment (jkschin) [reply _includes/cv/map.html:1]: or actually we could create a `table-cv-map` class and then assign the styles there in `_sass/base.scss`?

Comment (alshedivat) [reply _includes/cv/map.html:1]: yeah, let's create a separate class and assign it these styles.

**Comment by jkschin** [comment]
@alshedivat this still can't be merged as a maintainer needs to approve.

**Review by alshedivat — APPROVED** [review]
my apologies for the slow reply, really appreciate you contribution! going to merge now.

**Pull request merged**
